src/grammar.coffee

```diff
@@ -349,6 +351,48 @@ grammar =
     o 'CLASS SimpleAssignable EXTENDS Expression Block', -> new Class $2, $4, $5
   ]
 
+  Import: [
+    o 'IMPORT String',                          -> new Import null, $2
+    o 'IMPORT ImportClause FROM String',        -> new Import $2, $4
+  ]
+
+  ImportClause: [
+    o '{ }',                                    -> new ModuleList [], yes
+    o 'ModuleList OptComma',                    -> new ModuleList $1, no
```

Comment:
`a = { b, }` is a completely different case. I don't think `import a, from 'b'` should be allowed. CoffeeScript allows you to _remove_ commas here and there, not _add_ them.